### DISCUSSION TITLE
Dark colorblind theme is inaccessible for people with tritanomaly (purple vs blue)

### DISCUSSION THREAD

**Discussion by HappyRashair** [community/community#14864] [Accessibility] [answered]
These colors look almost the same to me:
![image](https://user-images.githubusercontent.com/95406820/163200921-e4621d50-c4d0-46bf-b7aa-23ee7ef31704.png)

**Comment by gaknoll** [comment, marked as answer]
@HappyRashair We have an all new Tritanopia colorblind theme shipping next week! Stay tuned 🎉

**Reply by gaknoll** [reply]
Also @HappyRashair since Tritanopia/Tritanomaly color blindness is pretty rare (and hard for us to do test internally) we'd REALLY appreciate all your feedback once it does ship! 🙏🏽

**Reply by jsoref** [reply]
@gaknoll: should this move from [🛣 on the roadmap](https://github.com/community/community/discussions?discussions_q=label%3A%22%F0%9F%9B%A3+on+the+roadmap%22) to [🚀 shipped](https://github.com/community/community/discussions?discussions_q=label%3A%22%F0%9F%9A%80+shipped%22)?

**Comment by HappyRashair** [comment]
Sure, I'll post the feedback as soon as I see new themes. Thanks for this :)

**Comment by HappyRashair** [comment]
New example, after theme change:
![image](https://user-images.githubusercontent.com/95406820/165085539-5eab2aff-c2dc-4ea9-a201-15603abad7c5.png)
Look really clear 😍 👍 

I'll give myself few days to look at it in various tabs and close this if everything is okay 😃

**Comment by HappyRashair** [comment]
It's been a while 😊 I really enjoy the new theme and I don't have any issues with it.
Thanks for great work 👍

**Comment by paauloriicardo669** [comment]
Fr5u7e5

Em qua, 27 de jul de 2022 10:09 PM, Josh Soref ***@***.***>
escreveu:
